codex-rs/app-server/src/bespoke_event_handling.rs

```diff
@@ -179,7 +179,7 @@ pub(crate) async fn apply_bespoke_event_handling(
             cwd,
             reason,
             risk,
-            allow_prefix: _allow_prefix,
+            proposed_execpolicy_amendment: _proposed_execpolicy_amendment,
```

Comment:
FYI, if you don't want to come up with a name:

```suggestion
            proposed_execpolicy_amendment: _,
```